test/BUILD.bazel

```diff
@@ -0,0 +1,50 @@
+load("@npm_bazel_karma//:index.bzl", "ts_web_test_suite")
+load("@npm_bazel_typescript//:defs.bzl", "ts_library")
+
+ts_library(
+  name = "test_lib",
+  deps = [
+    "//lib",
+    "@npm//@types",
+    "@npm//rxjs",
+  ],
+  srcs = glob([
+    "**/*.spec.ts",
+    "browser_entry_point.ts",
```

Comment:
Once we move this to angular/angular, buildifier will warn that entries in a glob with no wildcard should be outside the glob
`glob(["**/*.spec.ts"]) + ["browser-zone-setup.ts"]`

Reply:
Got it, I will modify this later, because currently this settings not work...